simulator/func_sim/alu.h

```diff
@@ -227,6 +227,7 @@ struct ALU
   
     // Generalized OR-Combine
     template<typename I> static void gorc( I* instr) { instr->v_dst[0] = gen_or_combine( instr->v_src[0], shamt_v_src2<typename I::RegisterUInt>( instr)); }
+    template<typename I> static void gorci( I* instr) { instr->v_dst[0] = gen_or_combine( instr->v_src[0], shamt_v_src2<typename I::RegisterUInt>( instr)); }
```

Comment:
Implementations of `gorc` and `gorci` match exaclty. What is the difference then?

Reply:
So I suppose that difference is only in the value of shamt_v_src2. 
In gorc it is the value of rs2 and in my case it is part of instruction

Reply:
Correct, but `shamt_v_src2` always takes data from `v_src2`, which you defined to be always zero
Are your tests passing? We cannot run it through CI since you have not updated submodule yet.
Please, if there are any infrastructure issues, don't hesitate to report. Testing is not an obstacle, but a help.

Reply:
I will update the submodule now, but tests on my pc are passing.
![yYmQfD7EvAU](https://user-images.githubusercontent.com/12381321/98017169-f04f3e80-1e0f-11eb-8679-034503fed10c.jpg)